demo/get_started.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting started with Jupytext\n",
+    "\n",
+    "This small notebook shows you how to activate Jupytext in the JupyterLab\n",
+    "environment. We'll show you a few things that you can do with Jupytext and\n",
+    "a bit of what happens under the hood.\n",
+    "\n",
+    "**Note: to run this notebook locally, you need to first follow the Jupytext\n",
+    "installation instructions and activate the JupyterLab plugin. If you're on\n",
+    "Binder, it should already work.**\n",
+    "\n",
+    "## Enabling Jupytext in a new notebook\n",
+    "\n",
+    "This notebook is brand new - it hasn't had any special extra metadata added\n",
+    "to it. \n",
+    "\n",
+    "If we want Jupytext to save files in multiple formats automatically,\n",
+    "we can use the JupyterLab **command palette** to do so.\n",
+    "\n",
+    "* Click on the little 🎨 icon to the left\n",
+    "* Then type **`Jupytext`**. You should see a number of commands come up. Each\n",
+    "  one tells Jupytext to save the notebook in a different\n",
+    "  file format automatically.\n",
+    "* Select **Pair notebook with Markdown**\n",
+    "\n",
+    "That's it! If you have Jupytext installed, it will now save your notebook in\n",
+    "markdown format automatically when you save this `.ipynb` file.\n",
+    "\n",
+    "After you've done this, save the notebook. You should now see a new file called\n",
+    "**`get_started.md`** in the same directory as this notebook.\n",
+    "\n",
+    "## How does Jupytext know to do this?\n",
+    "\n",
+    "Jupytext uses notebook-level metadata to keep track of what languages are paired\n",
+    "with a notebook. Below we'll print the metadata of this notebook so you can see\n",
+    "what the Jupytext metadata looks like."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import nbformat as nbf\n",
+    "from IPython.display import JSON\n",
+    "notebook = nbf.read('./get_started.ipynb', nbf.NO_CONVERT)\n",
+    "JSON(notebook['metadata'])"
+   ]
```

Comment:
Nice example! Thanks

Reply:
Chris, one meta-question... would you mind if we stored this notebook in its Markdown representation in the `demo` folder, and generate the `ipynb` in the postBuild with Jupytext? This way we would'nt need to review an ipynb file any more 😄

Reply:
ooh that's a cool idea - yeah sounds good

Reply:
More explicitly: I was thinking of adding these lines to `postBuild`:
```
cd demo
jupytext get_started.md --to ipynb --update-metadata '{"jupytext":null}'
rm get_started.md
```